sqlglot/dialects/snowflake.py

```diff
@@ -431,6 +431,7 @@ class Parser(parser.Parser):
                 this=seq_get(args, 0), expression=seq_get(args, 1), max_dist=seq_get(args, 2)
             ),
             "FLATTEN": exp.Explode.from_arg_list,
+            "FROM_HEX_BINARY": exp.Unhex.from_arg_list,
```

Comment:
This isn't a valid Snowflake function.

Reply:
Thanks for catching this, it was a copy paste error on my side.